subprojects/docs/src/docs/userguide/jvm/java_plugin.adoc

```diff
@@ -596,9 +598,8 @@ For instance, a `ServiceRegistryProcessor` could create a single `ServiceRegistr
 include::{snippetsPath}/java/incrementalAnnotationProcessing/groovy/processor/src/main/java/org/gradle/ServiceRegistryProcessor.java[tag=aggregating-annotation-processor]
 ----
 
-"Aggregating" processors have the following limitations:
+"Aggregating" processors have the following additional limitation:
 
-* They can only read `CLASS` or `RUNTIME` retention annotations
 * They can only read parameter names if the user passes the `-parameters` compiler argument.
```

Comment:
This also applies to all processors, for the same reason as the class/runtime retention annotations.

Reply:
Moved to `all` section